studio/components/interfaces/Support/SupportForm.tsx

```diff
@@ -443,6 +444,28 @@ const SupportForm = ({ setSentCategory }: SupportFormProps) => {
                         id="subject"
                         label="Subject"
                         placeholder="Summary of the problem you have"
+                        descriptionText={
+                          values.subject.length > 0 &&
+                          INCLUDE_DISCUSSIONS.includes(values.category) ? (
+                            <p className="flex items-center space-x-1">
+                              <span>Have you checked our </span>
+                              <Link
+                                key="gh-discussions"
+                                href={`https://github.com/orgs/supabase/discussions?discussions_q=${values.subject}`}
+                              >
+                                <a
+                                  target="_blank"
+                                  rel="noreferrer"
+                                  className="flex items-center space-x-2 text-scale-1000 underline hover:text-scale-1100 transition"
+                                >
+                                  Github discussions
+                                  <IconExternalLink size={14} strokeWidth={2} className="ml-1" />
+                                </a>
+                              </Link>
+                              <span>?</span>
```

Comment:
The question mark feels a little strange visually:
<img width="420" alt="Screenshot 2023-07-20 at 1 15 27 pm" src="https://github.com/supabase/supabase/assets/10985857/cde6ee0f-b9de-45eb-9412-8d130139a189">

Could consider changing it to something like "Check our GitHub Discussions to get an answer fast" for more active wording?

Reply:
Oh yeah agreed here - active voice